admin/controllers/service-manager.js

```diff
@@ -458,6 +460,114 @@ const controller = {
 
     req.flash('info', `'${school.name}' added as an MOD school`)
     return res.redirect(`/service-manager/mod-settings?hl=${school.urlSlug}`)
+  },
```

Comment:
I think it would be nice if the service-message methods could be moved to their own controller, perhaps called the `service-message.controller.js`

Reply:
@jon-shipley refactored to dedicated controller.